src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml

```diff
@@ -273,5 +273,26 @@
         <field-descriptor name="active" column="ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion"/>
         <field-descriptor name="lastUpdatedTimestamp" column="LAST_UPDT_TS" jdbc-type="TIMESTAMP" index="true"/>
     </class-descriptor>
-
+    
+    <class-descriptor class="edu.cornell.kfs.sys.businessobject.ISOFIPSCountryMap" table="CU_ISO_FIPS_CNTRY_MAP_T">
+        <field-descriptor name="isoCountryCode" column="ISO_POSTAL_CNTRY_CD" jdbc-type="VARCHAR" primarykey="true" indexed="true"/>
+        <field-descriptor name="fipsCountryCode" column="FIPS_POSTAL_CNTRY_CD" jdbc-type="VARCHAR" primarykey="true" indexed="true" />
+        <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" />
+        <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
+        <field-descriptor name="active" column="ACTV_IND" jdbc-type="varchar"
```

Comment:
You may want to uppercase "VARCHAR" for consistency.

Reply:
Fixing.